torchmultimodal/utils/assertion.py

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+from typing import Sized
+
+
+def assert_equal_lengths(*args: Sized, msg="iterable arguments must have same length."):
```

Comment:
Why cannot fit this into `common.py`? Or just a first step to split the ever growing `common.py`?

Reply:
yes, common was getting crowded and I think error checking utilities should have their own files